isotovideo

```diff
@@ -204,7 +204,8 @@ if (!$r && (my $nd = $bmwqemu::vars{NUMDISKS})) {
             $dir = 'assets_public';
         }
         next unless $name;
-        push @toextract, { hdd_num => $i, name => $name, dir => $dir };
+        my ($name,$format) = split /\./,$name;
```

Comment:
this is asking for trouble. the next one to put a . in its asset name will be F***Ked. the current regexp is much stricter